backend/src/main/java/com/google/step/data/Form.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+import java.util.Date;
+
+public class Form {
+    private Date date;
+    private long formId;
+    private String formName;
+
+    /**
+     * Key for the advertiser entity that this form belongs to.
+     * Transient to prevent GSON from serializing this when sending form data to client.
+     */
+    private transient Key advertiserKey;
+    private String googleKey;
+    private boolean verified;
+
+    /**
+     * Full parameter constructor for a Form object
+     * Auto generated the date object
+     * @param formId         the id of the form
+     * @param formName       the name of the form
+     * @param advertiserKey  the key of the advertiser entity in datastore
+     * @param googleKey      the google key for this form
+     * @param verified       whether or not this form has been verified
+     */
+    public Form (long formId,
+                 String formName,
+                 Key advertiserKey,
+                 String googleKey,
+                 boolean verified) {
+        this();
```

Comment:
You can also generate the getters and setters instead of initializing everything on the constructor.

IntelliJ has a tool that generate all that code automatically
https://www.jetbrains.com/help/idea/generating-code.html

Reply:
yeah, I'll remove the blank constructor